### DISCUSSION TITLE
Prevent reloading of specific loader after useFetcher submit

### DISCUSSION THREAD

**Discussion by roydigerhund** [remix-run/remix#2060] [Q&A] [answered]
Hi, I would like to prevent a specific loader from being reloaded after submitting a request with `useFetcher`.
I have a page with 12 images that are loaded in random order and users can bookmark them, which will trigger the `submit` function of `useFetcher`. This will request will reload the loading function of this route and thus fetching another set of 12 randomly selected images. But I want the images to not be refreshed.

Is there a way to bypass the reload here?

**Comment by merelinguist** [comment, marked as answer]
Either cache the loader response or use [`shouldReload`](https://remix.run/docs/en/v1/api/conventions#unstable_shouldreload).

**Reply by roydigerhund** [reply]
Thanks, `shouldReload` seems to work perfectly.
